docs/sources/shared/configuration.md

```diff
@@ -3775,6 +3775,10 @@ shard_streams:
 # CLI flag: -bloom-build.split-keyspace-by
 [bloom_split_series_keyspace_by: <int> | default = 256]
 
+# Experimental. Target chunk size in bytes for bloom tasks. Default is 100GB.
+# CLI flag: -bloom-build.target-chunk-size
+[bloom_task_target_chunk_size: <int> | default = 100GB]
```

Comment:
Not sure what a good default number would be here. Maybe 100GB is too much. What about 20GB? Wdyt?

Reply:
What's the impact of setting this too low/too high?

Reply:
But it wouldn't change the size of bloom blocks? If that's the case, overpartioning seems better than underpartitioning; 20GB seems fine and we can continue to adjust over time.